From 141910626b7601c4d697274ea49e951584ae24a5 Mon Sep 17 00:00:00 2001
From: Honggang Li <honli@redhat.com>
Date: Fri, 29 Dec 2017 17:21:52 +0800
Subject: [PATCH] srp_daemon: srp_daemon.service should be started after
 network.target

This is a rhel-7.5 specific workaround as rhel-7.5 is using
systemd-219.

The srp_daemon service will be started at the very beginning state
of systemd when boot/reboot the machine, in case srp_daemon.service
is not after network.target. As result, the srp_daemon.service will
be terminated because of SERVICE_FAILURE_RESOURCES.

Fixes: 1c7fe513e3e9 ("srp_daemon: One systemd service per port")
Signed-off-by: Honggang Li <honli@redhat.com>
---
 srp_daemon/srp_daemon.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/srp_daemon/srp_daemon.service.in b/srp_daemon/srp_daemon.service.in
index 188b7e1a..93e44425 100644
--- a/srp_daemon/srp_daemon.service.in
+++ b/srp_daemon/srp_daemon.service.in
@@ -3,6 +3,7 @@ Description=Daemon that discovers and logs in to SRP target systems
 Documentation=man:srp_daemon file:/etc/srp_daemon.conf
 DefaultDependencies=false
 Conflicts=emergency.target emergency.service
+After=network.target
 Before=remote-fs-pre.target
 
 [Service]
-- 
2.15.GIT

